src/Frontend/src/components/failedmessages/MessageView.vue

```diff
@@ -496,7 +496,7 @@ onUnmounted(() => {
                 retryMessage();
               "
               :heading="'Are you sure you want to retry this message?'"
-              :body="'Are you sure you want to retry this message?'"
+              :body="'Ensure that the selected message was not processed previously as this will create a duplicate message.'"
```

Comment:
I am a bit hesitant to change it to this.
The reason is that we want to help customers fall into the pit of success, so asking them to do this feels a bit wrong.
We should be doing all this to ensure they do not run into issues.

Reply:
@PhilBastian proposed to remove the body section of the dialog is body is not set